### PR TITLE
Updated by emmanueljet

### PR DESCRIPTION
**Pull request by emmanuelJet**

### CODE REVIEW

**Comment by emmanuelJet** [comment]
Faucet Image not showing on first commit fixed on the new commit

**Comment by emmanuelJet** [comment]
@eminogrande, @mpowaga, @dincho, and @noandrea kindly review my new commit on How to Get Aeternity Testnet Funds Tutorial.

**Comment by shekhar-shubhendu** [comment]
@emmanuelJet  please remove all the sections pertaining to `sdk-edgenet`

File: get-testnet-tokens.md [hunk 1]
```diff
@@ -36,23 +45,45 @@ aecli chain status -u https://sdk-testnet.aepps.com
...
 ```
+
 ### sdk-edgenet
+
 Audience – people developing the SDKs, developers who need the latest features from the develop branch on github. This network is used primarily for development and can be reset without notification.
 
 Epoch version checking:
 ```
-aecli chain status -u https://sdk-edgenet.aepps.com
+aecli chain status -u https://sdk-mainnet.aepps.com
```
Comment (thepiwo) [inline get-testnet-tokens.md]: use `sdk-testnet` for tutorials

Comment (emmanuelJet) [reply get-testnet-tokens.md]: Will do just that

**Comment by emmanuelJet** [comment]
> there is already a previous PR with similar changes, this will be prefered to merge. #34

Done

**Comment by emmanuelJet** [comment]
TUTORIAL: Deploying Sophia smart contracts with init parameters updated by @emmanuelJet. @thepiwo and @alekseytsekov kindly review.

AE Address: ak_rLoCtHE3NK9dKyCNonJFYWkEEfeAsDUWa887GsCKqV1rhSuT6

**Pull request closed without merging**